pom.xml

```diff
@@ -155,7 +155,7 @@
             <dependency>
                 <groupId>org.apache.kafka</groupId>
                 <artifactId>kafka-streams</artifactId>
-                <version>${kafka.version}</version>
+                <version>3.0.0-SNAPSHOT</version>
```

Comment:
This does not look like it's meant to be checked in.

Reply:
Ups...